USSF-ORBIT/ussf-portal-prototype
# chore: set up uswds sass dependencies

## Description 

Installs USWDS package and sets up Sass dependencies for use in Next.js and Storybook.

For Storybook:
*  Installs older versions of sass-loader, style-loader, and css-loader for Storybook for Webpack 4 compatibility
* Sample mixin to verify its working can be found in `Logo.module.scss` -- adds padding

For Next.js:
* Finagled import order so we can access dependencies inside css modules
* #TODO includes automating this import with webpack so we don't have to include `uswds-dependencies.scss` in every file

Fixes #19 

## Review Notes

To verify the setup is working:

* `yarn install` and `yarn dev`
  * Search bar component renders as expected
  * `container` class has additional padding from mixin (`Home.module.scss`)

* `yarn storybook`
  * `Logo` component container has additional padding from mixin (`Logo.module.scss`)
